### PR TITLE
[오승엽] vite설정

### PR DESCRIPTION
**Pull request by 5nxtnxtnxt**

### CODE REVIEW

File: front/.eslintrc.cjs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    'plugin:@typescript-eslint/recommended',
+    'plugin:react-hooks/recommended',
+  ],
+  ignorePatterns: ['dist', '.eslintrc.cjs'],
+  parser: '@typescript-eslint/parser',
+  plugins: ['react-refresh'],
+  rules: {
+    'react-refresh/only-export-components': [
+      'warn',
+      { allowConstantExport: true },
```
Comment (esthel7) [inline front/.eslintrc.cjs:15]: vite so good~~

File: front/README.md [hunk 2]
```diff
@@ -1,46 +1,27 @@
-# Getting Started with Create React App
+# React + TypeScript + Vite
```
Comment (kcwww) [inline front/README.md:1]: 좋네요

**Review by kcwww — APPROVED** [review]
LGTM BRO

**Review by esthel7 — APPROVED** [review]
LGTM~

**Pull request merged**